### ISSUE TITLE
Unable to change package name for the project

### ISSUE THREAD

**Issue by Drumiljani** [guardianproject/ChatSecureAndroid#711]
Change packagename of the application project.

After changing packagename, login stops working.

**Comment by n8fr8** [comment]
You need to also change all of the URI for the various services.

We are moving to a Gradle-build system which should make this easier moving forward.

**Comment by lassana** [comment]
@n8fr8 what services do you mean?

**Comment by AbdulSamadQureshi** [comment]
@n8fr , is that gradle -build system complete yet?

**Comment by n8fr8** [comment]
Yes, see github.com/zom and look at the ZomAndroid project. This is our
new project based on old chatsecure android code.

Also you might want to look at Conversations.im as an option.
